static/app/components/searchQueryBuilder/input.tsx

```diff
@@ -292,7 +269,7 @@ function SearchQueryBuilderInputInternal({
 
   return (
     <SearchQueryBuilderCombobox
-      items={items}
+      items={sections}
```

Comment:
We were originally sending in a flat list of items, but now are sending list of sections with child items. Below we are using a callback render function, which is why I needed to make the component props generic.